include/libdnf/comps/group/query.hpp

```diff
@@ -54,6 +54,15 @@ class GroupQuery : public libdnf::sack::Query<Group> {
         filter(F::name, pattern, cmp);
     }
 
+    /// Filter groups by packages they contain. Keep only groups that contain packages with given names.
+    ///
+    /// @param patterns         A vector of strings (package names) the filter is matched against.
+    /// @param cmp_type         A comparison (match) operator, defaults to `QueryCmp::EQ`.
+    ///                         Supported values: `EQ`, `NEQ`, `GLOB`, `NOT_GLOB`.
```

Comment:
In the usage from `group_info.cpp` we are using also `IGLOB`.

Reply:
I have removed all of the supported values because it supports many more operators and I don't think we should list them all for all the methods. They are described in the query_cmp.hpp.